tests/performance/README.md

```diff
@@ -31,7 +32,7 @@ example running commands for searching:
 ```
 ./build/tests/test_performance \
     '/data/random-100k-128-euclidean.hdf5' \
-    'search' \
+    'search:1' \
```

Comment:
@inabao add a new value k(or some other name) in the result json

Reply:
done